### PR TITLE
Populate tags

### PR DESCRIPTION
**Pull request by Civil**

### CODE REVIEW

File: expr/functions/holtWintersConfidenceBands/function.go [hunk 1]
```diff
@@ -31,48 +31,60 @@ func New(configFile string) []interfaces.FunctionMetadata {
...
-			ConsolidationFunc: arg.ConsolidationFunc,
-			XFilesFactor:      arg.XFilesFactor,
-			PathExpression:    fmt.Sprintf("holtWintersConfidenceLower(%s)", arg.Name),
-		}}
+		lowerSeries := types.MetricData{
+			FetchResponse: pb.FetchResponse{
+				Name:              fmt.Sprintf("holtWintersConfidenceLower(%s)", arg.Name),
+				Values:            lowerBand,
+				StepTime:          arg.StepTime,
+				StartTime:         arg.StartTime + 7*86400,
```
Comment (Felixoid) [inline expr/functions/holtWintersConfidenceBands/function.go:62]: Same here?

File: expr/functions/holtWintersForecast/function.go [hunk 2]
```diff
@@ -31,12 +31,12 @@ func New(configFile string) []interfaces.FunctionMetadata {
 }
 
 func (f *holtWintersForecast) Do(ctx context.Context, e parser.Expr, from, until int64, values map[parser.MetricRequest][]*types.MetricData) ([]*types.MetricData, error) {
```
Comment (Felixoid) [inline expr/functions/holtWintersForecast/function.go:33]: No bootstrapInterval here?

**Review by Felixoid — COMMENTED** [review]
Looks good, besides some points in bootstrapInterval calculation and missed argument in holtWintersForecast

File: expr/functions/holtWintersAberration/function.go [hunk 3]
```diff
@@ -65,16 +72,19 @@ func (f *holtWintersAberration) Do(ctx context.Context, e parser.Expr, from, unt
...
-			PathExpression:    fmt.Sprintf("holtWintersAberration(%s)", arg.Name),
-			ConsolidationFunc: arg.ConsolidationFunc,
-			XFilesFactor:      arg.XFilesFactor,
-		}}
+		r := types.MetricData{
+			FetchResponse: pb.FetchResponse{
+				Name:              fmt.Sprintf("holtWintersAberration(%s)", arg.Name),
+				Values:            aberration,
+				StepTime:          arg.StepTime,
+				StartTime:         arg.StartTime + 7*86400,
```
[the remark this answers is not in the document]
Comment (Civil) [reply expr/functions/holtWintersAberration/function.go:80]: Right, I've missed that part (and other 2 comments)

**Comment by Civil** [comment]
@Felixoid I've fixed the problems that you found.

File: expr/functions/holtWintersAberration/function.go [hunk 4]
```diff
@@ -43,12 +47,13 @@ func (f *holtWintersAberration) Do(ctx context.Context, e parser.Expr, from, unt
...
+	results := make([]*types.MetricData, 0, len(args))
 	for _, arg := range args {
 		var aberration []float64
 
 		stepTime := arg.StepTime
 
-		lowerBand, upperBand := holtwinters.HoltWintersConfidenceBands(arg.Values, stepTime, delta)
+		lowerBand, upperBand := holtwinters.HoltWintersConfidenceBands(arg.Values, stepTime, delta, bootstrapInterval/86400)
 
 		windowPoints := 7 * 86400 / stepTime
```
Comment (Felixoid) [inline expr/functions/holtWintersAberration/function.go:58]: Is it the same bootstrapInterval? =)

**Review by Felixoid — APPROVED** [review]
:+1:

**Pull request merged**